lib/cictl/index_command.rb

```diff
@@ -185,5 +178,30 @@ def index_deletes_for_date(date)
         logger.warn "could not find delfile '#{delfile}'"
       end
     end
+
+    def load_deleted_records
+      logger.info "Load most recent set of deleted records into solr"
+      if DeletedRecords.most_recent_non_empty_file
+        logger.info "Found #{DeletedRecords.most_recent_non_empty_file}"
+        solr_client.send_jsonl(DeletedRecords.most_recent_non_empty_file)
+      else
+        logger.error "Can't find any non_empty deleted_record files in #{DeletedRecords.save_directory}"
+      end
+    end
+
+    # Loads a full file and (assuming no exceptions were thrown)
+    # records that to the journal
+    def index_full_file(file)
+      logger.info "Using full marcfile #{file}"
+      # Calling the Thor "file" command.
+      call_file_command file
+
+      logger.info "Commit"
```

Comment:
Before, we were committing after loading deleted records, but not after we loaded the full file. We probably want to commit after we load the full file.